src/core/cmap.js

```diff
@@ -875,37 +873,37 @@ var CMapFactory = (function CMapFactoryClosure() {
       if (!isString(obj)) {
         break;
       }
-      var low = strToInt(obj);
+      const low = strToInt(obj);
       obj = lexer.getObj();
       if (!isString(obj)) {
         break;
       }
-      var high = strToInt(obj);
+      const high = strToInt(obj);
       cMap.addCodespaceRange(obj.length, low, high);
     }
     throw new FormatError("Invalid codespace range.");
   }
 
   function parseWMode(cMap, lexer) {
-    var obj = lexer.getObj();
+    const obj = lexer.getObj();
     if (Number.isInteger(obj)) {
       cMap.vertical = !!obj;
     }
   }
 
   function parseCMapName(cMap, lexer) {
-    var obj = lexer.getObj();
+    const obj = lexer.getObj();
     if (isName(obj) && isString(obj.name)) {
       cMap.name = obj.name;
     }
   }
 
   async function parseCMap(cMap, lexer, fetchBuiltInCMap, useCMap) {
-    var previous;
-    var embeddedUseCMap;
+    let previous;
+    let embeddedUseCMap;
```

Comment:
Nit: I *think* that you can change this into a one-liner, since the following should be allowed:
```suggestion
    let previous, embeddedUseCMap;
```